galaxy/Dockerfile

```diff
@@ -19,7 +19,7 @@ GALAXY_CONFIG_DIR=/etc/galaxy
 ENV GALAXY_CONFIG_FILE=$GALAXY_CONFIG_DIR/galaxy.ini \
 GALAXY_CONFIG_JOB_CONFIG_FILE=$GALAXY_CONFIG_DIR/job_conf.xml \
 GALAXY_CONFIG_JOB_METRICS_CONFIG_FILE=$GALAXY_CONFIG_DIR/job_metrics_conf.xml \
-GALAXY_VIRTUALENV=$GALAXY_ROOT/.venv/ \
+GALAXY_VIRTUAL_ENV=/galaxy_venv \
```

Comment:
So this is fine as is then I guess? Galaxy allows you to define a custom virtualenv path in job_conf.xml for the cluster nodes - and ideally a new a virtualenv would be created outside the container for the correct OS and path on the cluster (instead of just copying one over).